.github/workflows/kind-e2e.yaml

```diff
@@ -208,7 +208,7 @@ jobs:
 
     - name: Run tests
       run: |
-        make test-e2e-all
+        DAPR_E2E_TEST="workflows" make test-e2e-all
```

Comment:
Just leaving this comment here as a reminder that this needs to be reverted before merging :)